src/configure/configurers/provisioningConfigurer.ts

```diff
@@ -288,4 +293,13 @@ export class ProvisioningConfigurer implements IProvisioningConfigurer {
             files,
         } as DraftPipelineConfiguration;
     }
+
+    private async moveWorkflowFilesToRepoLocally(): Promise<void> {
+        const directoryPath: string = await this.localGitRepoHelper.getGitRootDirectory();
+        for (const file of this.filesToCommit) {
+            fse.moveSync(file.absPath, Path.join(directoryPath, file.path));
+            await vscode.window.showTextDocument(vscode.Uri.file(Path.join(directoryPath, file.path)), { preview: false });
```

Comment:
`Path.join(directoryPath, file.path)` is re-used. Please move to a local variable first.

Reply:
done